source/AI.cpp

```diff
@@ -391,13 +401,16 @@ void AI::UpdateKeys(PlayerInfo &player, Command &activeCommands)
 		IssueDeploy(player);
 
 	shared_ptr<Ship> target = flagship->GetTargetShip();
+	shared_ptr<Minable> targetAsteroid = flagship->GetTargetAsteroid();
 	Orders newOrders;
 	if(activeCommands.Has(Command::FIGHT) && target && !target->IsYours())
 	{
 		newOrders.type = target->IsDisabled() ? Orders::FINISH_OFF : Orders::ATTACK;
 		newOrders.target = target;
 		IssueOrders(player, newOrders, "focusing fire on \"" + target->Name() + "\".");
 	}
+	else if(activeCommands.Has(Command::FIGHT) && targetAsteroid)
```

Comment:
Don't we give a mining order for an asteroid, and not a fight order?

Reply:
It is F for fight my target.  Target varies between ship or asteroid.  You might be thinking of MINING command which is different; used for targeting.

Reply:
Okay got it, just wanted to clarify some stuff.
So mining just sets the target without it being set as your target to attack, correct? Still seems a bit weird to me but fine.

Reply:
I could rename it to TARGET_MINABLE if that's more clear.

V when pressed multiple times toggles harvest flotsam state.  If you press F to fight asteroids your fleet falls back to HARVEST so they collect the flotsam they just created